index.js

```diff
@@ -8,7 +8,7 @@ const isValidPackageName = str => {
   // <https://docs.npmjs.com/files/package.json#name>
 
   // ensure it's a string
-  if (typeof str !== 'string') return errors.notString;
+  if (!str || typeof str !== 'string') return errors.notString;
```

Comment:
We should instead use `is-string-and-not-blank` <https://github.com/niftylettuce/is-string-and-not-blank>